### DISCUSSION TITLE
Package "is not in this registry."

### DISCUSSION THREAD

**Discussion by o-t-w** [community/community#43565] [Actions] [answered]
### Select Topic Area

Question

### Body

I published my first GitHub Package using a GitHub action. 

I tried to NPM install it into a project. 
I am getting a 404 not found error and it says the package "is not in this registry". 

This is something I published only to the GitHub packages, not to the NPM registry.

**Comment by jongwooo** [comment, marked as answer]
@o-t-w Hi,

To install a package from GitHub Packages, you will need to include the `@<USERNAME>` namespace in the package name, as well as specify the URL of the GitHub Packages registry. For example:

```bash
npm install @<USERNAME>/<PACKAGE_NAME> --registry=https://npm.pkg.github.com
```

**Reply by Mller0** [reply]
npm install @<Moller>/<PACKAGE_NAME> --registry=https://npm.pkg.github.com
